homeassistant/components/august/__init__.py

```diff
@@ -78,7 +80,7 @@
 AUGUST_COMPONENTS = ["camera", "binary_sensor", "lock"]
 
 
-def request_configuration(hass, config, api, authenticator):
+def request_configuration(hass, config, api, authenticator, token_refresh_lock):
```

Comment:
Oh I didn't realize we still used the configurator 😱 That shit is so old lol. 

If you are looking into cleaning things up, run `python3 -m script.scaffold config_flow` and follow instructions 👍

Reply:
On the list :)